### ISSUE TITLE
Allow logging to syslog

### ISSUE THREAD

**Issue by andir** [strichliste/strichliste-backend#70]
In my installation I do not want anything in the webroot to be writable. The logs are currently written to `$projectDir/var/log` (or similar) whereas I would like them to end up in syslog so I can use my usual log collection, rotation, streaming, … setup without having to worry about strichliste. Another possible way might be just sending them through the standard PHP error mechanism which is already configured to pipe things to syslog (through the php-fpm settings).

(Opening this after an discussion with @schinken on IRC)

**Comment by schinken** [comment]
Any preferred logging format or events?

**Comment by schinken** [comment]
Ah you mean just the error log of symfony, right? Not events like "evt=boughtItem price=xxx userId=1232" etc

**Comment by andir** [comment]
Yes, just simply the error logs that currently vanish into somewhere (when there are not writeable files) :)

**Comment by schinken** [comment]
Okidoke. Im going to read some docs and try to change that!
